### PR TITLE
adding missing dependencies

### PR DESCRIPTION
**Pull request by mayankmahajan**
Was getting this issue:

> $ python run.py
> Traceback (most recent call last):
>   File "run.py", line 6, in <module>
>     from app import APP
>   File "<user.dir>/experiments/jvm-profiler/flamescope/app/__init__.py", line 34, in <module>
>     from app.views.flame_graph import MOD_FLAME_GRAPH
>   File "<user.dir>/experiments/jvm-profiler/flamescope/app/views/flame_graph.py", line 21, in <module>
>     from app.controllers.flame_graph import generate_flame_graph
>   File "<user.dir>/experiments/jvm-profiler/flamescope/app/controllers/flame_graph.py", line 22, in <module>
>     from app.perf.flame_graph import perf_generate_flame_graph
>   File "<user.dir>/experiments/jvm-profiler/flamescope/app/perf/flame_graph.py", line 25, in <module>
>     from app.common.fileutil import get_file
>   File "<user.dir>/experiments/jvm-profiler/flamescope/app/common/fileutil.py", line 22, in <module>
>     import magic
>   File "<user.dir>/.pyenv/versions/my-virtual-env-3.7.3/lib/python3.7/site-packages/magic.py", line 181, in <module>
>     raise ImportError('failed to find libmagic.  Check your installation')
> ImportError: failed to find libmagic.  Check your installation

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ protobuf==3.6.1
 python-magic==0.4.15
 six==1.12.0
 Werkzeug==0.14.1
+python-libmagic==0.4.0
```
Comment (spiermar) [inline requirements.txt]: Two different libraries. `python-magic==0.4.15` should suffice.

Comment (programwar) [reply requirements.txt]: I've had to use `python-magic-bin==0.4.14` to prevent this issue.

**Comment by dmatesic** [comment]
I have the same error (python v3.6.5). This fixed it for me: https://github.com/Yelp/elastalert/issues/1927#issuecomment-425040424

**Comment by spiermar** [comment]
`python-magic-bin` states that "This is a fork of the python-magic library but with binary packages for windows and mac.". Problem is not the python library, but you're missing `libmagic` on your system. It should be installed separately.
Not sure about `python-libmagic`, but seems abandoned. Closing the PR.

**Pull request closed without merging**